src/osd/OSD.cc

```diff
@@ -4332,20 +4415,6 @@ void OSD::RemoveWQ::_process(
 }
 // =========================================
 
-void OSD::do_mon_report()
-{
-  dout(7) << "do_mon_report" << dendl;
-
-  utime_t now(ceph_clock_now(cct));
-  last_mon_report = now;
-
-  // do any pending reports
-  send_alive();
```

Comment:
You seem to have ditched this call to send_alive but I can't find anything that replaced it in the same part of the git timeline.